kcards/views/api_rooms.py

```diff
@@ -45,7 +45,7 @@ def cleanup():
 
 @blueprint.route("/<code>")
 def detail(code):
-    room = Room.objects(code=code).first()
+    room = Room.objects(code=Room.clean_code(code)).first()
```

Comment:
I am not sure if this is the best place to do this nor if there are other places we'll need to cleanup incoming codes from the user.